<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimelinePage</class>
 <widget class="QWidget" name="TimelinePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>540</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-190</y>
        <width>315</width>
        <height>710</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="timeLineBox">
         <property name="title">
          <string>Timeline</string>
         </property>
         <layout class="QVBoxLayout" name="lay">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="lengthSizeLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string comment="Preferences">Timeline length:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="timelineLength">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="scrubBox">
            <property name="text">
             <string>Short scrub</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="emptyFrameBox">
         <property name="title">
          <string>Drawing</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelDrawingBehaviour">
            <property name="text">
             <string>When drawing on an empty frame:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonAddNewKey">
            <property name="statusTip">
             <string>Create a new (blank) key-frame and start drawing on it.</string>
            </property>
            <property name="text">
             <string>Create a new (blank) key-frame</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonDuplicate">
            <property name="statusTip">
             <string>Duplicate the previous key-frame and start drawing on the duplicate.</string>
            </property>
            <property name="text">
             <string>Duplicate the previous key-frame</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonDrawOnPrev">
            <property name="text">
             <string>Keep drawing on the previous key-frame</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAppliesTo">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(Applies to Pencil, Eraser, Pen, Polyline, Bucket and Brush tools)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="flipRollBox">
         <property name="title">
          <string>Flip and Roll</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" rowspan="2" colspan="2">
             <widget class="QLabel" name="flipRollNumDrawingsLanel">
              <property name="text">
               <string>Maximum numbers of drawings in roll</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QSlider" name="flipRollNumDrawingsSlider">
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QSlider" name="flipInBtwnMsecSlider">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>400</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="flipInBtwnMsecSpinBox">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>400</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLabel" name="flipInBtwnMsecLabel">
              <property name="text">
               <string>Msecs per drawing in flip inbetween</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="flipRollMsecsSpinBox">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>400</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1" rowspan="2">
             <widget class="QSpinBox" name="flipRollNumDrawingsSpinBox">
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QSlider" name="flipRollMsecsSlider">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>400</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="flipRollMsecsLabel">
              <property name="text">
               <string>Msecs per drawing in flip roll</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="soundScrubBox">
         <property name="title">
          <string>Sound scrub</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QSlider" name="soundScrubSlider">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="pageStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="sliderPosition">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="soundScrubSpinBox">
              <property name="suffix">
               <string> ms</string>
              </property>
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="layerVisibilityBox">
         <property name="title">
          <string>Layer Visibility</string>
         </property>
         <layout class="QVBoxLayout" name="layerVisibilityLayout">
          <item>
           <widget class="QLabel" name="labelVisibilityText">
            <property name="text">
             <string>Startup option</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="layerVisibilityComboBox">
            <item>
             <property name="text">
              <string>Current layer only</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Relative</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>All Layers</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelVisibility">
            <property name="text">
             <string>When layer visibility is relative (gray dot)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSlider" name="visibilitySlider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="visibilitySpinbox">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>timelineLength</tabstop>
  <tabstop>scrubBox</tabstop>
  <tabstop>radioButtonAddNewKey</tabstop>
  <tabstop>radioButtonDuplicate</tabstop>
  <tabstop>radioButtonDrawOnPrev</tabstop>
  <tabstop>flipRollNumDrawingsSlider</tabstop>
  <tabstop>flipRollNumDrawingsSpinBox</tabstop>
  <tabstop>flipRollMsecsSlider</tabstop>
  <tabstop>flipRollMsecsSpinBox</tabstop>
  <tabstop>flipInBtwnMsecSlider</tabstop>
  <tabstop>flipInBtwnMsecSpinBox</tabstop>
  <tabstop>layerVisibilityComboBox</tabstop>
  <tabstop>visibilitySlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
